sceptre/config/reader.py

```diff
@@ -569,3 +568,18 @@ def _parsed_stack_group_config(self, stack_group_config):
         }
         parsed_config.pop("stack_group_path")
         return parsed_config
+
+    def _collect_stack_tags(self, stack_name: str, config: dict) -> Dict[str, str]:
+        """
+        Assembles the stack tags dictionary, including default stack tags.
+
+        :param stack_name:
+        :param config:
+        :return:
+        """
+        stack_tags = {
+            'sceptre_name': stack_name,
+            'sceptre_project_code': config['project_code'],
```

Comment:
should we follow the cloudformation convention of using a `<prefix>:` ? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html

so something like `sceptre:name` and `sceptre:project-code`?

also, aws tags have limitations that we should check for? https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html